OmiseSDK/EContextInformationInputViewController.swift

```diff
@@ -48,6 +48,15 @@ class EContextInformationInputViewController: UIViewController, PaymentSourceCho
     @IBOutlet var gotoNextFieldBarButtonItem: UIBarButtonItem!
     @IBOutlet var doneEditingBarButtonItem: UIBarButtonItem!
     
+    override func loadView() {
+        super.loadView()
+        
+        view.backgroundColor = .background
+        formFieldsAccessoryView.barTintColor = .formAccessoryBarTintColor
+        
+        submitButton.defaultBackgroundColor = .omise
+        submitButton.disabledBackgroundColor = .line
```

Comment:
Could we create a new color for `.disabledBackgroudColor` for making it clear?

Reply:
Not sure if that worth it. We can have an alias to the same color instead

Reply:
Just beware when we change `.line` color.